code/modules/clothing/chameleon.dm

```diff
@@ -604,6 +639,8 @@
 	chameleon_action.chameleon_name = "Mask"
 	chameleon_action.chameleon_blacklist = typecacheof(/obj/item/clothing/mask/changeling, only_root_path = TRUE)
 	chameleon_action.initialize_disguises()
+	tongue_action = new(src)
+	tongue_action.generate_tongue_list()
```

Comment:
Does this need to be destroyed when the mask is destroyed or is this safely GC'd?

Reply:
Made it be set to null when the mask is destroyed.